asset_dashboard/forms.py

```diff
@@ -1,17 +1,30 @@
-from django.forms import ModelForm, CharField, TextInput, Textarea
-from .models import Project, Section, ProjectScore
+from django.forms import ModelForm, TextInput
+from .models import Project, ProjectScore, ProjectCategory, SenateDistrict, HouseDistrict, CommissionerDistrict, Zone
+from django.forms import inlineformset_factory
 
-class ProjectForm(ModelForm):
+
+class StyledFormMixin(object):
+    """
+    Generic mixin to consistently style each form field's html <input> element.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(StyledFormMixin, self).__init__(*args, **kwargs)
```

Comment:
it's a bit better to do this as

```
super().__init__(*args, **kwargs)
```

that way we don't need to change anything if decide we want to change the class name.

Reply:
fixed with 9bc9149b0f23a29567f413cb4d3d3ac6b6ff69ec